src/content/docs/journey-performance/improve-website-performance.mdx

```diff
@@ -0,0 +1,231 @@
+---
+title: "Improve your website's performance"
+---
+import browserCoreWebVitals from 'images/browser_diagram_core-web-vitals.png'
+
+import browserBrowserSummary from 'images/browser_screenshot-full_browser-summary.png'
+
+import browserErrorOverview from 'images/browser_screenshot-full_error-overview.png'
+
+import browserSuggestedServiceLevels from 'images/suggested_service_levels.png'
+
+A good website—you know it when you surf it, right? It's so fast you never notice the load time. It's available whenever you need it. And, of course, it's error-free. But _measuring_ your site's performance, let alone improving it, is a tricky task.
+
+This tutorial walks you through how to use New Relic to improve your site's experience. You'll start by "instrumenting" your site, which means adding some JavaScript code that measures the actual end-user experience for each and every visit. From there, you'll learn how to understand how your site is performing today, where you can improve, and how to keep an eye on things in future.
+
+## Improve your website's performance
+
+<Steps>
+
+<Step>
+### Instrument your site
+
+Before you can improve your site's performance, you need to see what's going on. New Relic monitoring always starts with an **integration**, a piece of code that collects data from a technology you care about and reports it to New Relic. 
+
+To monitor the front end of your website, the integration you want is our **Browser monitoring agent**. This is a small snippet of JavaScript that you'll include in your HTML `<head>`, very similar to Google Analytics or other scripts. 
```

Comment:
```suggestion
To monitor the front end of your website, the integration you want is our **Browser monitoring agent**. This is a small snippet of JavaScript that you'll include in your HTML `<head>`, similar to adding Google Analytics or other scripts to your site. 
```

Reply:
Changed the structure of this sentence a bit, thanks for the comment!